src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A class for Deadlines tasks.
+ */
+public class Deadlines extends Task {
```

Comment:
Perhaps the name of this class can be named "Deadline" instead? Since an instance of "Deadlines" is referring to a singular deadline.